src/mutation-observers/selector_observer.ts

```diff
@@ -19,14 +19,14 @@ export class SelectorObserver implements AttributeObserverDelegate, ElementObser
   constructor(
     element: Element,
     attributeName: string,
-    scope: Element,
+    controllerElement: Element,
```

Comment:
I think it's switched up now. `controllerElement` is now `document.body` and `element` the actual controller element which is used for observing the attribute